articles/quickstart/native/flutter/interactive.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Add login to your Flutter app
+default: true
+description: This tutorial demonstrates how to add user login with Auth0 to a Flutter application using the Auth0 Flutter SDK
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - flutter
+  - dart
+  - ios
+  - android
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/build
+  - files/main
+---
+
+# Add login to your Flutter app
+
+Auth0 allows you to quickly add authentication and access user profile information in your application. This guide demonstrates how to integrate Auth0 with a Flutter application using the [Auth0 Flutter SDK](https://github.com/auth0/auth0-flutter).
+
+## Getting started
+
+This quickstart assumes you already have a [Flutter](https://flutter.dev/) application up and running. If not, check out the [Flutter "getting started" guides](https://docs.flutter.dev/get-started/install) to get started with a simple app, or clone [our sample apps](https://github.com/auth0-samples/auth0-flutter-samples).
```

Comment:
> or clone [our sample apps](https://github.com/auth0-samples/auth0-flutter-samples).

Isn't this a bit redundant, since they'll have the option to download the sample app?
Also, this can be a bit confusing, since the sample app already has Auth0 integrated, so it's not really a suitable 'Getting started' candidate (it's not a blank slate).